src/dune_rules/melange_rules.ml

```diff
@@ -139,6 +133,21 @@ let add_rules_for_entries ~sctx ~dir ~expander ~dir_contents ~scope
         build_js ~dir ~loc:(Some loc) ~pkg_name ~module_system:mel.module_system
           ~dst_dir ~obj_dir ~sctx ~build_dir ~lib_deps_js_includes m)
   in
+  let* () =
+    match mel.alias with
+    | None -> Memo.return ()
+    | Some alias_name ->
+      let alias = Alias.make alias_name ~dir in
+      let* build_paths =
+        Memo.parallel_map
+          (Modules.fold_no_vlib modules ~init:[] ~f:(fun x acc -> x :: acc))
```

Comment:
I am not sure about the perf implications of this. We just iterate over the list of entries / modules a few lines above, but it seemed good to separate js rule vs alias rule building.